src/macaron/__main__.py

```diff
@@ -221,6 +222,26 @@ def main(argv: list[str] | None = None) -> None:
         If ``argv`` is ``None``, argparse automatically looks at ``sys.argv``.
         Hence, we set ``argv = None`` by default.
     """
+    # Handle presence of tokens.
+    token_file = "./.macaron_env_file"  # nosec B105
+    if os.path.exists(token_file):
+        with open(token_file, encoding="utf-8") as file:
+            for line in file:
+                print(f"Line: {line}")
+                if not line or "=" not in line:
+                    continue
+                key, value = line.rstrip().split("=")
```

Comment:
This will throw ValueError on unpack if there is more than one `=`.
Should change it to `split("=", 1)` so that it always splits on the first `=` just in case a value containing additional `=` characters is ever valid.